digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/DataFixture.scala

```diff
@@ -479,6 +500,61 @@ object DataFixture {
     println("\n")
   }
 
+  def checkBusStopMatchingBetweenOTHandTR(): Unit = {
+    println("\nVerify if OTH mass transit stop exist in Tierekisteri, if not present, create them. ")
+    println(DateTime.now())
+
+    var persistedStop: Seq[PersistedMassTransitStop] = Seq()
+    var missedBusStopsOTH: Seq[PersistedMassTransitStop] = Seq()
+
+    //Get a List of All Bus Stops present in Tierekisteri
+    val busStopsTR = tierekisteriClient.fetchActiveMassTransitStops
+
+    //Save Tierekisteri LiviIDs into a List
+    val liviIdsListTR = busStopsTR.map(_.liviId)
+
+    //Get All Municipalities
+    val municipalities: Seq[Int] =
+      OracleDatabase.withDynSession {
+        Queries.getMunicipalities
+      }
+
+    municipalities.foreach { municipality =>
+      println("Start processing municipality %d".format(municipality))
+
+      //Get all OTH Bus Stops By Municipality
+      persistedStop = massTransitStopService.getByMunicipality(municipality, true)
+
+      persistedStop.foreach { stop =>
+        // Validate if OTH stop are known in Tierekisteri and if is maintained by ELY
+        if (!liviIdsListTR.exists(_ == "OTHJ%d".format(stop.nationalId)) && massTransitStopService.isStoredInTierekisteri(Some(stop))) {
```

Comment:
Not all stops are of that form - some stops have older LiviIDs which are saved in stops' properties.